test/helpers/cons.go

```diff
@@ -197,6 +197,7 @@ const (
 	segmentationFault = "segmentation fault"        // from https://github.com/cilium/cilium/issues/3233
 	NACKreceived      = "NACK received for version" // from https://github.com/cilium/cilium/issues/4003
 	RunInitFailed     = "JoinEP: "                  // from https://github.com/cilium/cilium/pull/5052
+	sizeMismatch      = "size-mismatch"             // from https://github.com/cilium/cilium/issues/7851
```

Comment:
I was unclear in my issue description :/ sorry about that. This should be a search for "size mismatch for BPF map" instead:

https://github.com/cilium/cilium/blob/7881196da7be4d8b90458098f9cdcdbc5f3d0296/pkg/bpf/bpf_linux.go#L316

https://github.com/cilium/cilium/blob/7881196da7be4d8b90458098f9cdcdbc5f3d0296/pkg/bpf/bpf_linux.go#L324

Reply:
We should search for https://github.com/cilium/cilium/blob/7881196da7be4d8b90458098f9cdcdbc5f3d0296/pkg/bpf/bpf_linux.go#L339 and https://github.com/cilium/cilium/blob/7881196da7be4d8b90458098f9cdcdbc5f3d0296/pkg/bpf/bpf_linux.go#L332.

Reply:
Welp, I was wrong (again)! Sorry for the churn, @eloycoto - thanks for clarifying, @brb .